salt/_modules/containerd.py

```diff
@@ -26,6 +26,7 @@ def load_cri_image(path):
         Path of the Docker image archive to load
     """
     log.info('Importing image from "%s" into CRI cache', path)
-    return __salt__["cmd.run_all"](
-        'ctr --debug -n k8s.io image import "{0}"'.format(path)
-    )
+    cmd = 'ctr --debug -n k8s.io image import "{}"'.format(path)
+    if fullname:
+        cmd += ' --index-name "{}"'.format(fullname)
```

Comment:
Why is this needed whilst it wasn't before?

Reply:
It's to import an index name for the image, it was not needed before because we used the "repositories" file in the tar archive
```json
# cat repositories
{"k8s.gcr.io/pause":{"3.2":"a38657cdc544bd8c100ef952fc60e31709509b0bc4f31804ab107fe7fd0e6f4a"}}
```

So several options:
1- At build time we put the right repo in the tar for the image (which mean we "compute" the repository name at build time, where today it's at run time in salt)
2- At run time we create the tar file containing the right `repositories` file with the repo (it's a bit of work...)
3- We use `--index-name` so that in the cache we have an index on the pause image

I choose the last option as it was the simplest to do, but we can still change this if we need to

Reply:
OK, thanks for explaining. Maybe consider putting such intel in commit message in the future :)

Reply:
Right